edward/inferences.py

```diff
@@ -208,6 +214,10 @@ def initialize(self, n_iter=1000, n_data=None, n_print=100,
         init = tf.initialize_all_variables()
         init.run()
 
+        # Start input enqueue threads.
```

Comment:
If the user is in the setting of preloaded data or reading from files, it could still be useful to continue using the same threads after interrupting computation. So let's not do my proposal above.